src/dom.js

```diff
@@ -112,6 +112,11 @@ export function documentContainsPolyfillInternal_(doc, element) {
  */
 export function removeElement(element) {
   if (element.parentElement) {
+    const activeElement = element.ownerDocument.activeElement;
+    // Restore focus to document if the focused element is inside the removed one.
+    if (element == activeElement || element.contains(activeElement)) {
```

Comment:
I don't think most caller can know whether to set that boolean or not. That requires them knowing for sure the focus is not in the element they are removing, but focus is controlled by the user.

In terms of expensiveness, I have no idea how much this would effect the overall performance. My guess is negligible but let's get @dvoytenko and @cramforce thoughts here.